external/mmdetection/detection_tasks/apis/detection/model_wrappers/instance_segmentation.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
you need to modify deploy method so that this model will appear in the deployed package

Reply:
done